api/v1alpha4/azurecluster_validation.go

```diff
@@ -38,6 +38,8 @@ const (
 	// described in https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules
 	subnetRegex       = `^[-\w\._]+$`
 	loadBalancerRegex = `^[-\w\._]+$`
+	// MaxLoadBalancerOutboundIPs is the maximum number of outbound IPs in a Standard LoadBalancer frontend configuration
+	MaxLoadBalancerOutboundIPs = 16
```

Comment:
This number comes from [here](https://github.com/Azure/aks-engine/blob/f81f85fd9f8a3e83607c883204417685ddc2020e/pkg/api/common/const.go#L36)